amundsen_application/static/js/components/common/PopularTables/index.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+
+// TODO: Use css-modules instead of 'import'
+import './styles.scss';
+
+import { POPULAR_TABLES_LABEL, POPULAR_TABLES_INFO_TEXT, POPULAR_TABLES_SOURCE_NAME } from './constants';
+import InfoButton from 'components/common/InfoButton';
+import SearchList from 'components/SearchPage/SearchList';
+
+import { getPopularTables } from 'ducks/popularTables/reducer';
+import { GetPopularTablesRequest, TableResource } from 'ducks/popularTables/types';
+import { GlobalState } from 'ducks/rootReducer';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+export interface StateFromProps {
+  popularTables: TableResource[];
+}
+
+export interface DispatchFromProps {
+  getPopularTables: () => GetPopularTablesRequest;
+}
+
+export type PopularTablesProps = StateFromProps & DispatchFromProps;
+
+export class PopularTables extends React.Component<PopularTablesProps> {
+  public static defaultProps: Partial<PopularTablesProps> = {};
```

Comment:
Probably don't need `defaultProps` if there are none.

The `container` will always pass `popularTables` as a prop, as `globalState.popularTables` is initialized to an empty array.
```
{
    popularTables: state.popularTables,
  };
```

Reply:
+1